src/server/main.cpp

```diff
@@ -28,7 +29,12 @@ int main(int argc, char** argv) {
             level = static_cast<LevelKey>(result["level"].as<int>());
         }
 
-        ServerGameController server_game_controller(level);
+        bool tcp_port_opt_present = result.count("port");
+        uint32_t tcp_port = GAME_SETTINGS.SERVER_TCP_PORT;
+        if (tcp_port_opt_present) {
```

Comment:
ditto
we dont need this var, put the `.count` straight in the if

Reply:
I think it's more readable naming hwy we're getting the count personally